src/SpendPermissions.sol

```diff
@@ -150,33 +147,20 @@ contract SpendPermissions {
         emit RecurringAllowanceRevoked(hash, recurringAllowance.account, recurringAllowance);
     }
 
-    /// @notice Withdraw tokens using a recurring allowance and approval signature.
-    ///
-    /// @dev Convenience function for offchain preparation from apps that have an ERC-7715 permissions context.
-    ///
-    /// @param context Flat bytes value representing an approved recurring allowance.
-    /// @param recipient Address to withdraw tokens to.
-    /// @param value Amount of token attempting to withdraw (wei).
-    function withdraw(bytes calldata context, address recipient, uint160 value) external {
```

Comment:
if there was feedback that led to the decision to remove the overloading of this function I haven't seen it yet, but I do think it simplifies things without leading to any obvious negative implications for ux so seems like a good idea to me